### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by lindalu-MSFT**

### CODE REVIEW

File: docs/overview/script-storage.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To find who is using Office Scripts with the search tool, add `.osts` in the **F
 
 ## See also
 
-- [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
+- [Sharing Office Scripts in Excel for the web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
```
Comment (AlexJerabek) [inline docs/overview/script-storage.md:45]: This is matching the casing on the linked article. I think we should leave it as is.

```suggestion
- [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
```

**Review by AlexJerabek — APPROVED** [review]
Approving assuming the link text comment is addressed.

**Comment by AlexJerabek** [comment]
Also, @lindalu-MSFT, should this be merged into the master branch instead of a second patch branch?

**Comment by lindalu-MSFT** [comment]
Long story but yes, indeed the goal is to merge fixes in both branches to master!

**Comment by AlexJerabek** [comment]
@lindalu-MSFT Okay cool, just checking!

**Pull request merged**